packages/loader/driver-definitions/src/urlResolver.ts

```diff
@@ -38,6 +38,26 @@ export interface IUrlResolver {
     ): Promise<string>;
 }
 
+/**
+* Information that can be returned by a lightweight, seperately exported driver function. Used to preanalyze a URL
+* for driver compatibility and preload information.
+*/
+export interface DriverPreCheckInfo {
+    /**
+     * Only needed as long as long as Loader.request() does not work as intended. When Loader.request() caches and
+     * resolves pathing properly, this can be removed. #4489, #4491
+     */
+    componentPath: string;
```

Comment:
we shouldn't used the term component here, maybe containerPather

Reply:
I think odsp driver already uses datastorepath for this?

Reply:
Hmm, both seem better (although this field should hopefully be temporary). I can go with containerPath for now, since that's closest to what it is.

Reply:
should also be optional if it will be remove, maybe even pre-mark as deprecated

Reply:
It is still required until #4489 and #4491 are resolved. Premarked it as deprecated though